### ISSUE TITLE
更新右键桌面更换壁纸选项以及自动更换壁纸功能建议

### ISSUE THREAD

**Issue by InkBlackScorpion** [rocksdanister/lively#1293]
非常需要这两个功能

**Comment by rocksdanister** [comment]
Is this what you mean?

<img width="335" alt="image" src="https://user-images.githubusercontent.com/17554161/178029917-d8655762-506c-4d2e-a7e1-f501d7bff52a.png">

**Comment by InkBlackScorpion** [comment]
> Is this what you mean?
> 
> 
> 
> <img width="335" alt="image" src="https://user-images.githubusercontent.com/17554161/178029917-d8655762-506c-4d2e-a7e1-f501d7bff52a.png">
> 
> 

No, this function is similar to the effect of right clicking the icon on the taskbar
[image](https://user-images.githubusercontent.com/100986616/178033172-2db4b4e9-e59e-47aa-8b69-9f232988170a.png)

**Comment by InkBlackScorpion** [comment]
> Is this what you mean?
> 
> 
> 
> <img width="335" alt="image" src="https://user-images.githubusercontent.com/17554161/178029917-d8655762-506c-4d2e-a7e1-f501d7bff52a.png">
> 
> 

It also needs the function of automatically changing wallpaper regularly

**Comment by rocksdanister** [comment]
![image](https://user-images.githubusercontent.com/17554161/178034268-98ea0849-d240-40a0-974b-070e844d3c1a.png)

System-tray menu has this:
<img width="361" alt="image" src="https://user-images.githubusercontent.com/17554161/178034141-6a9d80c6-7af6-4637-989d-a978000e251e.png">

**Comment by ecr-oha-lime** [comment]
> 

Windows有一个定时自动切换壁纸的功能。希望lively可以拥有这项功能。
========translate=======
Windows has a function that automatically switches wallpapers at regular intervals. Hope lively can have this feature.

**Comment by rocksdanister** [comment]
> Windows有一个定时自动切换壁纸的功能。希望lively可以拥有这项功能。
========translate=======
Windows has a function that automatically switches wallpapers at regular intervals. Hope lively can have this feature.


https://github.com/rocksdanister/lively/issues/137

**Comment by alittledudu** [comment]
也许吧
